src/controller/cve.controller/index.js

```diff
@@ -259,15 +235,15 @@ router.put('/cve/:id',
     required: true,
     content: {
       "application/json": {
-        schema:{ $ref: '/schemas/cve/cve-record-minimum-request.json' }
+        schema:{ $ref: '/schemas/cve/create-full-cve-record-request.json' }
       }
     }
   }
   #swagger.responses[200] = {
-    description: 'Returns the CVE record updated.',
+    description: 'The CVE Record updated.',
```

Comment:
Remove period from end. Also, The updated CVE Record would be slightly better wording